docs/internal/dev_guidelines.rst

```diff
@@ -53,6 +53,36 @@ Dependencies
 
   :doc:`/devguide/building/dependencies` for all dependencies and details on how these are used in OpenDDS.
 
+*********************
+Dependency Management
+*********************
+
+ACE/TAO
+=======
+
+By default, the :ghfile:`configure` script and :ref:`CMake support<cmake-building>` use released versions of ACE/TAO.
+The same versions are used for pull requests.
+The versions are documented in :ghfile:`acetao.ini`.
+Currently, the :ghfile:`configure` scripts uses a release of ACE6 and CMake uses a release of ACE7.
+
+The versions are updated by the GitHub Actions workflow in :ghfile:`.github/workflows/update-ace-tao.yml`.
+When a new version of ACE/TAO is available, the workflow creates a pull request based on the new versions.
```

Comment:
This might or might not be worth mentioning here, but the update ACE/TAO workflow currently only automatically updates the version if it's a micro/patch release. Other releases require changing the `version` property and running the release script manually.